john555/true-space
# Task/Create sign in and sign up pages.

This PR adds sign up and sign in pages.

## How to test
- Checkout to this branch using `git pull && git checkout latest-branch`
- Start the app `php artisan serve`
- The `/account/create` and `/account/signin` routes should contain the sign-up and sign in routes respectively.
## Issues
closes #7 
closes #8
